charts/refarch/Chart.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v2
+name: refarch
+description: Helm Chart for deploying a it@M Reference Architecture application.
+type: application
+version: 1.0.0
+home: https://github.com/it-at-m/helm-charts/tree/main/charts/refarch
+icon: https://assets.muenchen.de/logos/itm/itm-logo-256.png
+sources:
+  - "https://github.com/it-at-m/helm-charts"
+dependencies:
+  - name: backend
+    version: 1.0.0
+  - name: webcomponent
+    version: 1.0.0
+  - name: frontend
+    version: 1.0.0
+  - name: refarch-gateway
+    version: 1.4.4
+    repository: "@it-at-m"
+maintainers:
+  - name: devtobi
+    url: https://github.com/devtobi
+  - name: simonhir
+    url: https://github.com/simonhir
+  - name: DanielOber
+    url: https://github.com/DanielOber
+  - name: langehm
+    url: https://github.com/langehm
```

Comment:
warum macht jetzt das @it-at-m/refarch  das Helm-Chart? Ich habe gedacht, dass ist Aufgabe des @it-at-m/sps . Meiner Meinung nach muss man bei Erstellung so viel beachten. Das Thema ist hochkomplex

Reply:
I agree with @hupling that we have to discuss who is responsible.